pontoon/pretranslation/tests/test_pretranslate.py

```diff
@@ -622,8 +617,8 @@ def test_get_pretranslations_fluent_multiline(
     # 100% TM match exists
     tm = TranslationMemoryFactory.create(
         entity=entity_b,
-        source="Multi Line Message",
-        target="TM: Multi Line Message",
+        source="Multi\nLine\nMessage",
+        target="TM: Multi\nLine\nMessage",
```

Comment:
In actual use, we do have multiline values in the translation memory, and that's why we need separate `tm_source` and `gt_source` values in `Pretranslate.pattern()`.

Previously, values with newlines were never matched to translation memory entries.